Geometry/MuonNumbering/src/GEMNumberingScheme.cc

```diff
@@ -59,31 +59,44 @@ int GEMNumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) {
   //  ring    = num.getSuperNo(theRingLevel);
   // GEM are only on the first ring
   ring = 1;
-  station = num.getSuperNo(theStationLevel);
+
+  // GE0 has the layer encoded in the ring level
+  if (num.getBaseNo(theRingLevel) == 0) {  // 0 => GE1/1, GE2/1
```

Comment:
is there a relevant constant for this?

Reply:
I don't think so. The ring level isn't used in GEM so its always 0 (as mentioned in the (old) comments a little above), so I took it over since I needed to encode the GE0 layer in the new xmls. I don't think we can change the naming, since 'mg_ring' is used in all the other xmls. @bsunanda do you have any thoughts on this?